api-server/test/events.e2e-spec.ts

```diff
@@ -285,14 +286,6 @@ describe('Events API', () => {
         expect(pageSize).toEqual(20);
         expect(events.length).toEqual(allVerifiedEvents.length);
 
-        const expected = allVerifiedEvents.map((ve) => {
-          return { id: ve.id, title: ve.title };
-        });
-
-        const actaul = events.map((e) => {
-          return { id: e.id, title: e.title, date_times: e.date_times };
-        });
-
         for (let i = 0; i < numEventsWithDateTimes; i++) {
```

Comment:
^ Small cleanup, these were unused, deleting them to get warnings out of my IDE :-)